zephyr/CMakeLists.txt

```diff
@@ -235,29 +235,39 @@ zephyr_system_include_directories(../examples/platforms)
 # Determine which libs should be linked in
 set(ot_libs "")
 
+if(CONFIG_OPENTHREAD_COPROCESSOR_RCP)
```

Comment:
This does not seem to be needed, the CLI libs will only be linked when shell is enabled (which is blocked for coprocessor now).

Reply:
Yes. Applied.